waitsensor/waitsensor.go

```diff
@@ -425,9 +411,14 @@ func (cs *counter) run(ctx context.Context) error {
 			// process for each stream in the list of cameras
 			totalCounts := 0
 			for camName, bbs := range cs.validRegions {
-				img, release, err := streams[camName].Next(ctx)
+				// We ignore the metadata returned with the bytes
+				imgBytes, _, err := cs.cams[camName].Image(ctx, utils.MimeTypePNG, nil)
+				if err != nil {
+					return errors.Errorf("camera %v error retrieving bytes: %q", camName, err)
+				}
+				img, err := png.Decode(bytes.NewReader(imgBytes))
```

Comment:
There is a convenience function @randhid made to grab and decode the image for Go, it is in RDK in the camera.go file, do not decode the image yourself